### DISCUSSION TITLE
Discussion of Alpha Build

### DISCUSSION THREAD

**Discussion by exedox** [Inpacchi/Manga-Tagger#18] [Q&A]
I wanted to leave this open for discussing all your new builds that aren't necessarily related to bugs directly.  This is just for general questions I'll have:

- I noticed that the config has the new windows version created but it doesn't seem to be super updated with the latest info about the executable file.  It's still in progress right?
- I'd like to confirm if I'm using it correctly.  To have the exe file open prior to downloading new chapters with FMD.  Is that right? Does that mean tagger does not need a service to work?
- I'm curious about your roadmap for this project.  Of course you don't have to share it because I don't want to take up too much of your time away from developing it, but wanted to see what ideas you were going to do with improving it. 

Thanks again for doing all this!

**Comment by Inpacchi** [comment]
1. What config are you referring to?
2. With the current v1.1.1-alpha build, yes, Manga Tagger must be open prior to downloading new chapters. When I release the executable for v1.1.2-alpha, it will scan for new downloads upon start-up. Manga Tagger should still be configured as a service, however I am still working on building the actual installer to do that automatically.
3. For roadmap, [look here](https://github.com/Inpacchi/Manga-Tagger/projects/1). No worries, I don't mind sharing :)

Currently my priorities are working on Linux support as I have to test on a Linux distro, as well as developing command line functions so you can run it as a service or a utility.

After that, next priorities are cover downloading and volume creation, as well as a background updater service so you don't have to pull down a new executable each time.

**Comment by exedox** [comment]
1. It's the settings and configuration documentation for the Windows version (which looks pretty much like how the original set up was.) I assume it's a placeholder for when you come out with an official build of the exe file.

Nice! I've been wanting some sort of linux support.  May I suggest creating a docker file so then it would be able to run nicely on any distro? A docker would be super helpful in that regard.   Thanks for the prompt response :D.  Looking forward to testing it out.

**Comment by Ablex222** [comment]
Just wanted to say thanks for being receptive to feed back. I am very much looking forward to the day cover art is immediately available. I also had a couple other ideas and wanted to propose them. 
1. Would it be possible that during the metadata creation, the software does not only check one source for genre information, but populates that with the results of both?
2. I was looking at anilist/myanimelist and noticed that there are tags for manga. Would it be possible to include the tags from the metadata sources in the metadata? From what I can tell, there doesn't seem to be a way to save tags in comicinfo.xml, but I was thinking including them in Genre would be effective enough. Probably would be best as a toggleable feature however, since I feel it could be a personal preference thing.
3. Kind of a weird one, but would it be possible to have the software keep a running list of the names of series it has processed in a text document along with their other names? Would be useful for when trying to audit collections. 
4. Another weird one, it would be great if multiple instances could be run. I can see benefits to being able to run multiple instances with each being specified to specific folders to process. This may be addressed with docker implementation however, as long as the port to access doesn't matter. 
5. A GUI for the purpose of renaming files that failed to be named correctly. Doesn't have to be full fledged, just something like a executable that asks for a file path to the misprocessed series and a list of options generated by the algorithm.
6. Proxy integration. It is not really a concern currently since the entire system can run through one in a VM. However, there is something to be said about not wanted to get IP blocked on personal connections, especially since the tool already can easily increase scaling.   

I can't help but feel like I am giving you homework lol. Thanks again for all your work.

**Reply by Ablex222** [reply]
Just going to add one more thing to this Wishlist. The ability for the software to include as the first lines of the summary, the manga's ratings from the 2 sites if available. So something like this....

MAL = 7.8/10 
Anilist = 7/10
Description..................................

**Comment by Inpacchi** [comment]
@exedox @Ablex222 check out https://github.com/Inpacchi/Manga-Tagger/discussions/23

**Comment by exedox** [comment]
Hi @Inpacchi It's been a while, but I wanted to touch base with you on how to use the new Alpha Build.  The documentation hasn't been done yet, although I do remember just executing it in the directory where we want to place the stuff.  I wanted to see if we still needed all the same prereqs as before.

I went ahead and tested what I remembered us discussing and nice touch on letting us go for the directory on where FMD is.  If we were to edit the settings to change the directories of where we want our library to be in, and where the downloads are, would it break anything?

I noticed there isn't a function for adding metadata to existing library contents yet.  Is it in the works, or is this something that's a bit lower on the priority list.

**Comment by aeryl1999** [comment]
does the alpha still require mongodb and compass?
